
<!-- Lauch the gazebo sim or connect to Sawyer-->
<!-- Launch Joint Action trajectory server -->
<!-- Launch Sawyer MoveIt config -->

<launch>

  <node name="traj_with_sawyer" pkg="the_mighty_sawyer" type="traj_with_sawyer" output="screen"/>

</launch>
